src/plugins/keyframes-plugin.js

```diff
@@ -9,11 +9,30 @@ export default function keyframesPlugin(
   const newStyle = Object.keys(style).reduce(
     (newStyleInProgress, key) => {
       let value = style[key];
-      if (key === 'animationName' && value && value.__radiumKeyframes) {
-        const keyframesValue = (value: Keyframes);
-        const {animationName, css} = keyframesValue.__process(config.userAgent);
-        addCSS(css);
-        value = animationName;
+      if (
+        key === 'animationName' &&
+        value &&
+        (value.__radiumKeyframes || Array.isArray(value))
+      ) {
+        if (Array.isArray(value)) {
+          value = value
+            .map(v => {
+              const keyframesValue = (v: Keyframes);
+              const {animationName, css} = keyframesValue.__process(
+                config.userAgent,
+              );
+              addCSS(css);
+              return animationName;
+            })
+            .join(', ');
+        } else {
+          const keyframesValue = (value: Keyframes);
+          const {animationName, css} = keyframesValue.__process(
+            config.userAgent,
+          );
+          addCSS(css);
+          value = animationName;
```

Comment:
can we reduce the duplicate logic here and extract this to a shared method?

ie:

```
const processKeyframeStyle = (value) => {
  const keyframesValue = (value: Keyframes);
  const { animationName, css } = keyframesValue.__process(config.userAgent);
  addCSS(css);
  return animationName;
}
```

```
if (Array.isArray(value)) {
  value = value.map(processKeyframeStyle).join(', ');
} else {
  value = processKeyframeStyle(value);
}
```